flutter/shell/platform/tizen/nui_autofill_popup.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2023 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef EMBEDDER_NUI_AUTOFILL_POPUP_H_
+#define EMBEDDER_NUI_AUTOFILL_POPUP_H_
+
+#include <dali-toolkit/devel-api/controls/popup/popup.h>
+
+#include <functional>
+
+namespace flutter {
+
+using OnCommit = std::function<void(std::string str)>;
+using OnRendering = std::function<void()>;
+
+class NuiAutofillPopup : public Dali::ConnectionTracker {
+ public:
+  void Hide();
+
+  void Hidden();
+
+  void OutsideTouched();
+
+  void PrepareAutofill();
+
+  void PopupAutofill(Dali::Actor* actor);
+
+  void SetOnCommit(OnCommit callback) { on_commit_ = callback; }
+
+  bool Touched(Dali::Actor actor, const Dali::TouchEvent& event);
```

Comment:
The callbacks can be made private.

Reply:
Moved `Hidden`, `OutsideTouched`, `Touched` to private region.

Reply:
`Prepare` could also be private.

Reply:
Moved `Prepare` to private region. thank you.